include/lsst/afw/image/Filter.h

```diff
@@ -90,6 +92,14 @@ class FilterProperty {
      * Return the filter's effective wavelength (nm)
      */
     double getLambdaEff() const { return _lambdaEff; }
+    /**
+     * Return the filter's minimum wavelength (nm)
```

Comment:
Is it possible to be more precise here?

For example, are these the values beyond which < X% of the total throughput exists?  or where the fractional throughput first drops below X% of the peak?  After all, if we're being really pedantic, I think we expect some light leaks (at the ~10^-5 of peak throughput level) across pretty much the entire visible range from 350nm to 1100nm (but obviously it isn't useful to set lambdaMin=350 and lambdaMax=1100 for every filter).  If you do have a concrete procedure for calculating these, or even if the answer is "I eyeballed it" it would be good to document that.